src/containers/forms/PlanForm/helpers.ts

```diff
@@ -145,21 +147,21 @@ export function extractActivityForForm(activityObj: PlanActivity): PlanActivityF
     goalDescription: activityObj.goal.description || '',
     goalDue:
       activityObj.goal.target[0].due && activityObj.goal.target[0].due !== ''
-        ? moment(activityObj.goal.target[0].due).toDate()
+        ? parseISO(`${activityObj.goal.target[0].due}${DEFAULT_TIME}`)
```

Comment:
We switched to using parseISO to avoid dealing with timezone problems. i.e. the fact that dates from OpenSRP dont include time meant that the time element before now was in the timezone of the local machine, which brought inconsistencies.

Reply:
Ooooooh great catch! 👏